multivariate-linear-regression/utils.js

```diff
@@ -49,7 +45,7 @@ export const loadCsv = async (filename) => {
   return new Promise(resolve => {
     const url = `${BASE_URL}${filename}.csv`;
 
-    console.log(`  * Downloading from: ${url}`);
+    console.log(`  * Downloading data from: ${url}`);
```

Comment:
This was a debug statement. It can be removed now or we can update `status` accordingly.